src/analysis/play-utilities/PlayDecider.cpp

```diff
@@ -6,5 +6,18 @@
 
 namespace rtt::ai::analysis {
 // Maybe plays should have some sort of score based on the world how good they are
-Play PlayDecider::decideBestPlay(world::World *world, world::Field *field, std::vector<Play> validPlays) { return Play(); }
+Play* PlayDecider::decideBestPlay(world::World* world, const world::Field& field, std::vector<Play*> const& validPlays) {
+    int max = -1;
+    if (validPlays.empty()) {
+        std::cerr << "valid plays are empty" << std::endl;
+    }
+    for (auto& play : validPlays) {
+        int temp = play->scorePlay(world, field);
+        if (temp > max) {
+            max = temp;
+        }
```

Comment:
This can be in 1 line using max = std::max(max, temp). I prefer shorter code over longer code